src/lib.rs

```diff
@@ -165,23 +172,22 @@ pub trait PolynomialCommitment<F: Field>: Sized {
         labeled_polynomials: impl IntoIterator<Item = &'a LabeledPolynomial<F>>,
         commitments: impl IntoIterator<Item = &'a LabeledCommitment<Self::Commitment>>,
         point: F,
-        opening_challenge: F,
         rands: impl IntoIterator<Item = &'a LabeledRandomness<Self::Randomness>>,
         rng: Option<&mut dyn RngCore>,
+        fs_rng: &mut Self::RandomOracle,
```

Comment:
I'd consider open as a high-level function, which internally takes care of the Fiat-Shamir transform in its own way. 
(Otherwise why having the function at all, if it just wraps ``open_individual_opening_challenges()``?)
Hence, I recommend to remove the ``fs_rng`` here, and absorb the statement of the opening proof (i.e. the commitments and the point) internally before calling ``open_individual_opening_challenges()``.

Reply:
Done